robot.py

```diff
@@ -23,6 +29,16 @@ def teleopPeriodic(self):
             self.drive_joystick.getRawAxis(0), squaredInputs=True)
         self.drivetrain.forward_at(self.drive_joystick.getRawAxis(1))
 
+        action = IntakeAction.Stop
+
+        if self.drive_joystick.getRawButton(1):
+            action = IntakeAction.Outtake
+
+        if self.drive_joystick.getRawButton(2):
+            action = IntakeAction.Intake
+
+        self.intake.cur_action = action
```

Comment:
Rather than setting a variable in the class, can you make methods in the class for:
- `intake`
- `outtake`